packages/devtools/lib/src/utils.dart

```diff
@@ -145,6 +145,10 @@ String longestFittingSubstring(
   return originalText.substring(0, i);
 }
 
+/// Whether a given code unit is a letter (A-Z or a-z).
+bool isLetter(int codeUnit) =>
```

Comment:
I'm not sure if there's anything we can leverage in Dart or the core libraries here? cc @scheglov